### PR TITLE
[IMP] mail_tracking_mailgun: link to bounce event

### PR DESCRIPTION
**Pull request by chienandalu**
So it's easier to follow events

cc @Tecnativa

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Thanks! :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please forward-port it to v12

**Comment by sergio-teruel** [comment]
!! Fast !! :smile:

**Comment by pedrobaeza** [comment]
Oh, wait, I have removed the branch for not merging.

This will force to re-translate everything. Please put the link part on the other side, as this doesn't need to be translated.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
See previous comment

**Comment by chienandalu** [comment]
good catch

**Comment by pedrobaeza** [comment]
Please change this

File: mail_tracking_mailgun/models/res_partner.py [hunk 1]
```diff
@@ -26,10 +26,13 @@ def _email_bounced_set(self, reason, event):
...
-            body = _('Email has been bounced: %s\n'
-                     'Reason: %s\n'
-                     'Event: %s') % (partner.email, reason,
-                                     event and event.id or _('unknown'))
+            body = _(
+                'Email has been bounced: %s\n'
+                'Reason: %s\n'
+                'Event: '
+                '<a href=# data-oe-model=mail.tracking.event data-oe-id=%d>'
+                '%s</a>') % (partner.email, reason, event and event.id or 0,
```
Comment (yajo) [inline mail_tracking_mailgun/models/res_partner.py]: Apart from what @pedrobaeza said, also here you can use:

```suggestion
                '%s</a>') % (partner.email, reason, event.id or 0,
```

File: mail_tracking_mailgun/models/res_partner.py [hunk 2]
```diff
@@ -26,10 +26,13 @@ def _email_bounced_set(self, reason, event):
...
+                             event and event.id or _('unknown'))
```
Comment (yajo) [inline mail_tracking_mailgun/models/res_partner.py]:
```suggestion
                             event.id or _('unknown'))
```

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@pedrobaeza Changed

**Review by pedrobaeza — APPROVED** [review]

**Review by Tardo — APPROVED** [review]
LGTM :+1:

**Review by yajo — APPROVED** [review]

File: mail_tracking_mailgun/models/res_partner.py (hunk 2 above)
Comment (chienandalu) [reply mail_tracking_mailgun/models/res_partner.py]: This is why the https://travis-ci.org/OCA/social/jobs/582600826#L1289 :)

Fixed in the last commit anyway

**Comment by chienandalu** [comment]
merge again, please

**Pull request merged**